### PR TITLE
fix(console): add json validator to connector config

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add jsonValidator to connector config field.

## Testing

截屏2022-12-30 下午12 01 47

### CODE REVIEW

File: packages/console/src/components/CodeEditor/index.tsx [hunk 1]
```diff
@@ -49,49 +53,52 @@ const CodeEditor = ({
...
+            }}
+            language={language}
+            style={theme}
+          >
+            {value}
+          </SyntaxHighlighter>
+        </div>
       </div>
-    </div>
+      {hasError && errorMessage && <div className={styles.errorMessage}>{errorMessage}</div>}
```
Comment (gao-sun) [inline packages/console/src/components/CodeEditor/index.tsx:100]: nit: the error message can be more friendly? like the exact position of error
(let's talk to designer and do it later)

File: packages/console/src/utilities/validator.ts [hunk 2]
```diff
@@ -16,3 +16,13 @@ export const uriOriginValidator = (value: string) => {
     return false;
   }
 };
+
+export const jsonValidator = (value: string) => {
```
Comment (gao-sun) [inline packages/console/src/utilities/validator.ts:20]: maybe we can use `trySafe` from essentials>

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/components/CodeEditor/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/components/CodeEditor/index.tsx:100]: OK

File: packages/console/src/utilities/validator.ts (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/utilities/validator.ts:20]: It seems not easy to return `true` or `false` by using `trySafe`, given that `try...catch` is used in many validators, I created an issue to track. (LOG-5003)

**Pull request merged**